ogcore/txfunc.py

```diff
@@ -688,11 +693,23 @@ def txfunc_est(
         wsse = wsse_cstr
         params = mono_interp
         params_to_plot = params
+    elif tax_func_type == "mono2D":
+        mono_interp, _, wsse_cstr, _, _ = monotone_spline(
+            df[["total_labinc", "total_capinc"]].values,
+            df["etr"].values,
```

Comment:
This case should use `txrates` rather than `df["etr"]` since lines above adjust `txrates` for the specific rate chosen (i.e., ETR, MTRx, MTRy).

Also, for consistency with other cases, will want to use `X` and `Y` rather than columns straight from `df` here - same for the weights.

Reply:
Think your PR to my fork fixed this